app/views/states.py

```diff
@@ -0,0 +1,122 @@
+from flask import Blueprint, request, jsonify, make_response
+
+from app.models import State
+
+states_bp = Blueprint('states_bp', __name__)
+
+
+@states_bp.route('/states', methods=['GET'])
+def get_states():
+    state = State()
+    states = state.get_states()
+    # return jsonify({"message": "states retrieved successfully", "status": 200, "result": states}), 200
+    return make_response({"message": "states retrieved successfully", "status": 200, "result": states},200)
+
+
+@states_bp.route('/create', methods=['POST'])
+def store():
+    data = request.get_json()
+    name = data.get("name")
+    abbreviation = data.get("abbreviation")
+    capital = data.get("capital")
+    population = data.get("population")
+    year_admitted = data.get("year_admitted")
+
+    state = State()
+
+    result = state.save(name, abbreviation, capital, population, year_admitted)
+    if result['status'] == 409:
+        return make_response({
+            "message": result['message'],
+            "status": result['status'],
+            "data": result['result']
+        }, 409)
+    return make_response({
+        "message": result['message'],
+        "status": result['status'],
+        "data": result['result']
+    }, 201) 
+    # return jsonify({"result": result['result'], 'message': result['message'], 'status': result['status']})
```

Comment:
Can you redo the logic here? Status code `409` is great for conflicting data yes but how?